scripts/sns/install/upgrade_dapp.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script is used to create an SNS proposal for increasing the cost of proposal rejection to 200 MOD
+# This script is called by upgrade_test_canister.sh
+# For detailed explanation please see the GitHub Repo: https://github.com/dfinity/sns-testing/tree/main
+
+
+set -euo pipefail
+
+CURRENTDIR="$(pwd)"
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+REPODIR="$(pwd)"
```

Comment:
The variable name is confusing to me.  After `cd -- "$(dirname -- "${BASH_SOURCE[0]}")"` , you should be in the same folder as this script.
Suggest to use SCRIPT_DIR here.